apps/passport-client/components/screens/FrogScreens/ManageFrogsSection.tsx

```diff
@@ -0,0 +1,338 @@
+import {
+  CredentialManager,
+  FrogCryptoFrogData,
+  isFrogCryptoFrogData,
+  requestFrogCryptoDeleteFrogs,
+  requestFrogCryptoUpdateFrogs
+} from "@pcd/passport-interface";
+import { Separator } from "@pcd/passport-ui";
+import { SerializedPCD } from "@pcd/pcd-types";
+import _ from "lodash";
+import { Dispatch, SetStateAction, useEffect, useMemo, useState } from "react";
+import Table from "react-table-lite";
+import styled from "styled-components";
+import { appConfig } from "../../../src/appConfig";
+import {
+  useCredentialCache,
+  useIdentity,
+  usePCDCollection
+} from "../../../src/appHooks";
+import { ErrorMessage } from "../../core/error";
+import { useAdminError } from "./useAdminError";
+
+export function ManageFrogsSection() {
+  const [newFrogs, setNewFrogs] = useState<FrogCryptoFrogData[]>([]);
+  const [newFrogsError, setNewFrogsError] = useState<string>();
+  const onTextChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    try {
+      const parsedData = frogParser(event.target.value);
+      setNewFrogs(parsedData);
+      setNewFrogsError(undefined);
+    } catch (error) {
+      setNewFrogsError(error.message);
+    }
+  };
+
+  const {
+    result: frogs,
+    isLoading,
+    updateFrogs,
+    deleteFrogs,
+    error
+  } = useFrogs();
+
+  const [selectedFrogIds, setSelectedFrogIds] = useState<number[]>([]);
+  useEffect(() => {
+    setSelectedFrogIds([]);
+  }, [frogs]);
+
+  useAdminError(error);
+
+  return (
+    <>
+      <h2>Add New Frogs</h2>
+      <p>
+        Go to the Export tab of the Frog data spreadsheet, click on Export JSON
+        in the toolbar, and copy the generated JSON representation of frogs.
```

Comment:
maybe link it here + on the other page that references the sheet. we should make it private if you do this.